src/page.ts

```diff
@@ -452,6 +452,22 @@ export class Page extends ExtendedEventEmitter implements InnerLogger {
     return this.mainFrame().focus(selector, options);
   }
 
+  async textContent(selector: string, options?: types.TimeoutOptions): Promise<string | null> {
+    return this.mainFrame().textContent(selector, options);
+  }
+
+  async innerText(selector: string, options?: types.TimeoutOptions): Promise<string | null> {
```

Comment:
How can innerText or innerHTML be null?

Also, for textContent, would it be much more convenient to convert null into empty string?

Reply:
these all are actually always strings; casted.